x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/prebuilt_rule_import.md

```diff
@@ -151,58 +167,65 @@ When the user imports the rule
 Then the import should be rejected with a message "rule_id field is required"
 ```
 
-#### Scenario: Importing a prebuilt rule with a matching rule_id but missing a version field
+#### Scenario: Importing a prebuilt rule with a matching `rule_id` but missing a `version` field
 
 **Automation**: 1 integration test.
 
 ```Gherkin
 Given the import payload contains a prebuilt rule without a version field
+And its rule_id matches one or a few rule assets from the installed package
 When the user imports the rule
 Then the import should be rejected with a message "version field is required"
 ```
 
-#### Scenario: Importing an existing custom rule missing a version field
+#### Scenario: Importing an existing custom rule missing a `version` field
 
 **Automation**: 1 integration test.
 
 ```Gherkin
-Given the import payload contains an existing custom rule without a version field
+Given the import payload contains a custom rule without a version field
+And its rule_id does NOT match any rule assets from the installed package
+And this custom rule has already been created
 When the user imports the rule
 Then the rule should be updated
 And the ruleSource type should be "internal"
 And the "version" field should be set to the existing rule's "version"
 ```
 
-#### Scenario: Importing a new custom rule missing a version field
+#### Scenario: Importing a new custom rule missing a `version` field
 
 **Automation**: 1 integration test.
 
 ```Gherkin
-Given the import payload contains a new custom rule without a version field
+Given the import payload contains a custom rule without a version field
+And its rule_id does NOT match any rule assets from the installed package
+And this custom rule hasn't been created yet
 When the user imports the rule
 Then the rule should be created
 And the ruleSource type should be "internal"
 And the "version" field should be set to 1
 ```
 
-#### Scenario: Importing a rule with overwrite flag set to true
+#### Scenario: Importing a rule with `overwrite` flag set to true
```

Comment:
The two scenarios for the `overwrite` flag are not related to the logic of importing prebuilt rules. This is a common logic for importing any rules. Let's extract these into a separate test plan.

We could just add `And the overwrite flag is set to true` to most of the scenarios in this plan.

Reply:
PR: https://github.com/elastic/kibana/pull/213434